lifecycle/manager.go

```diff
@@ -107,3 +107,8 @@ func (lcs *LifecycleServicesManager) RemoteDeleteReleaseBundle(params distributi
 	rbService := lifecycle.NewReleaseBundlesService(lcs.config.GetServiceDetails(), lcs.client)
 	return rbService.RemoteDeleteReleaseBundle(params, dryRun)
 }
+
+func (lcs *LifecycleServicesManager) ExportReleaseBundle(releaseBundleExportParams *lifecycle.ReleaseBundleExportParams, queryParams lifecycle.CommonOptionalQueryParams) (exportResponse lifecycle.ReleaseBundleExportedStatusResponse, err error) {
```

Comment:
I believe it will be best to follow the convention where the release bundle details are first in the service signature, separated from other params.
In addition, I suggest not use a pointer to the params if we don't need the reference. Rather to pass by value.

```suggestion
func (lcs *LifecycleServicesManager) ExportReleaseBundle(rbDetails lifecycle.ReleaseBundleDetails, releaseBundleExportParams lifecycle.ReleaseBundleExportParams, queryParams lifecycle.CommonOptionalQueryParams) (exportResponse lifecycle.ReleaseBundleExportedStatusResponse, err error) {
```